index.html

```diff
@@ -2,194 +2,225 @@
 layout: home
 title: Bazel - a fast, scalable, multi-language and extensible build system"
 ---
-
-<div class="hero">
-  <div class="container">
-    <img src="{{site_root}}images/bazel-logo.svg"
-         title="Bazel"
-         alt="Bazel logo"
-         class="logo img-responsive center-block" />
-    <p class="lead">{Fast, Correct} - Choose two</p>
-  </div>
-</div>
-<div class="hero-bar">
-  <div class="container">
-    <a class="btn btn-lg"
-       href="{{ "/docs/install.html" | prepend: site_root }}">
-       Install Bazel</a>
-    <a class="btn btn-success btn-lg"
-       href="{{ "/docs/getting-started.html" | prepend: site_root }}">
-       Get Started</a>
-  </div>
-</div>
-<div class="landing-feature-1">
-  <div class="container">
-    <div class="row">
-      <div class="col-sm-12">
-        <h1>Correct, reproducible, fast builds for everyone</h1>
-        <p class="lead">Build software of any size, quickly and reliably, just as engineers do at Google.</p>
-        <p>Bazel is Google's own build tool, now publicly available in Beta.
-           Bazel has built-in support for building both client and server software, including client
-           applications for both Android and iOS platforms. It also provides an extensible framework
-           that you can use to develop your own build rules.
-        </p>
+<div class="home">
+  <div class="landing-section hero">
+    <div class="container">
+      <div class="row">
+        <div class="col-md-6">
+          <img src="{{site_root}}images/bazel-logo.svg"
+               title="Bazel"
+               alt="Bazel logo"
+               class="logo img-responsive center-block" />
+          <p class="tagline">{Fast, Correct} - Choose two</p>
+        </div>
+        <div class="col-md-6">
+          <h1>Build and test software of any size, quickly and reliably</h1>
```

Comment:
The use of bold text for this title seems a bit out of place. How would it look if this used the same thinner font weight as the other headings on the website?